src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/border/border_kernel_ref.cpp

```diff
@@ -35,6 +45,14 @@ ParamsKey BorderKernelRef::GetSupportedKey() const {
     k.EnableOutputLayout(DataLayout::b_fs_yx_fsv16);
     k.EnableOutputLayout(DataLayout::b_fs_yx_fsv32);
     k.EnableOutputLayout(DataLayout::b_fs_zyx_fsv16);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv4_fsv2);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv4_fsv4);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv8_fsv2);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv8_fsv4);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv16_fsv16);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv32_fsv16);
+    k.EnableOutputLayout(DataLayout::bs_fs_yx_bsv32_fsv32);
+    k.EnableOutputLayout(DataLayout::bs_fs_zyx_bsv16_fsv16);
```

Comment:
isn't it possible just to enable all layouts?

Reply:
You are right. Updated to use EnableAll[Input/Output]Layout.